cache.py

```diff
@@ -47,4 +52,4 @@ def clear(self):
     def get_video_id(url) -> str:
         parsed_url = urlparse(url)
         video_id = parse_qs(parsed_url.query)['v'][0]
-        return(video_id)
+        return(video_id)
```

Comment:
undo this change, just leave a blank line at the bottom of this file, under the return statement

Reply:
done